samples/components/dialects/Hive12Dialect.tdd

```diff
@@ -1,7 +1,10 @@
-<dialect name='HiveDialectSDK'
-         base='HiveDialect'
-         class='hive_clone'
-         version='18.1'>
+<!--
```

Comment:
I think you can just remove this comment, we're tracking it internally, just creates confusion in the interim.